dir.props

```diff
@@ -43,6 +43,12 @@
     <ToolsDir Condition="'$(ToolsDir)'==''">$(PackagesDir)Microsoft.DotNet.BuildTools.$(BuildToolsVersion)/lib/</ToolsDir>
   </PropertyGroup>
 
+  <!-- Test runtime -->
+  <PropertyGroup>
```

Comment:
They are overriding [properties that buildtools uses](https://github.com/dotnet/buildtools/blob/b668d329ae4950ba1500cc4a8d480188123856cb/src/Microsoft.DotNet.Build.Tasks/Targets/publishtest.targets).